### DISCUSSION TITLE
Zoominfo Bootcamp

### DISCUSSION THREAD

**Discussion by OrManor** [ReadyCodePush/cohort-0x01#19] [General]
‏פיד הייטקס: יש פה ג׳וניורים תותחים שרוצים להשתתף בבוטקאמפ של Zoominfo?
חברה מעולה, אנשים אחלה, והזדמנות ממש טובה להשתלב
https://t.co/k4jK0vSA44‎

דברו אלי.
‎#zi ‎#zoominfo

https://www.zoominfo.com/about/careers?gh_jid=6004914002&gh_src=d14a9e1e2

**Comment by lirantal** [comment]
היי אור, נשמע מגניב ממש!
אני חושב שיכול להיות רלוונטי מאוד ל @elladvir

(בקשה קלה: את יכולה לתקן בבקשה את הלינק הראשון ששיתפת? הוא לא עובד :))

**Reply by OrManor** [reply]
Sure

**Reply by elladvir** [reply]
Thanks, I just started a new job couple of month ago, but thanks for the invite :-)

**Reply by lirantal** [reply]
That's fine.
I think the Bootcamp thing is scheduled for the summer or something which is several months out, and also it's more about helping you level up and learn, and not only about getting a job. If I'm not mistaken, a job position isn't even guaranteed.
